app/src/utils/constants.ts

```diff
@@ -41,7 +41,7 @@ export const DefaultForageConfig: LocalForageConfig = {
   // this store should be used for any on-chain/off-chain data but never user data (as we might clear it without notice)
   name: 'dGrants',
   // we can bump this version number to bust the users cache
-  version: 1,
+  version: 3,
```

Comment:
Why 3 and not 2?

Reply:
I wasn't sure what should I put, thought maybe some other PR is going to use 2.

Reply:
There are no other open PRs changing this, so we should use 2 🙂
```suggestion
  version: 2,
```

Reply:
fixed